src/coreComponents/constitutive/permeability/ProppantPermeability.hpp

```diff
@@ -84,9 +84,10 @@ class ProppantPermeabilityUpdate : public PermeabilityBaseUpdate
                                                              localIndex const q,
                                                              real64 const & oldHydraulicAperture,
                                                              real64 const & newHydraulicAperture,
+                                                             real64 const & dHydraulicAperture_dNormalJump,
                                                              real64 const & proppantPackVolumeFraction ) const override final
   {
-    GEOS_UNUSED_VAR( q );
+    GEOS_UNUSED_VAR( q, dHydraulicAperture_dNormalJump );
 
```

Comment:
Not related to your modification. But I just noticed that the proppant permeability needs to be corrected too. 
(a^3)/12 -> (a^2)/12

Reply:
Let's do this separately